$OpenBSD: patch-tmac_doc-common,v 1.5 2013/07/13 07:39:00 schwarze Exp $
chunk 1: Keep traditional OpenBSD page headers (local change).
chunk 2: default value for the .Os macro (local change).
chunk 3: More OpenBSD release versions (to be submitted upstream).
chunk 4: Handle .Dd Mdocdate (local change).
chunk 5: Do not use an ugly double dash in the NAME line (local change).
--- tmac/doc-common.orig	Thu Feb  7 13:06:08 2013
+++ tmac/doc-common	Mon Jul  8 07:09:20 2013
@@ -231,17 +231,17 @@
 .
 .\" an alternative, more detailed scheme for naming the manual sections
 .\"
-.ds doc-volume-operating-system BSD
+.ds doc-volume-operating-system OpenBSD
 .nr doc-volume-operating-system-ateol 0
-.ds doc-volume-ds-1 General Commands Manual
-.ds doc-volume-ds-2 System Calls Manual
-.ds doc-volume-ds-3 Library Functions Manual
-.ds doc-volume-ds-4 Kernel Interfaces Manual
-.ds doc-volume-ds-5 File Formats Manual
-.ds doc-volume-ds-6 Games Manual
-.ds doc-volume-ds-7 Miscellaneous Information Manual
+.ds doc-volume-ds-1 Reference Manual
+.ds doc-volume-ds-2 Programmer's Manual
+.ds doc-volume-ds-3 Programmer's Manual
+.ds doc-volume-ds-4 Programmer's Manual
+.ds doc-volume-ds-5 Programmer's Manual
+.ds doc-volume-ds-6 Reference Manual
+.ds doc-volume-ds-7 Reference Manual
 .ds doc-volume-ds-8 System Manager's Manual
-.ds doc-volume-ds-9 Kernel Developer's Manual
+.ds doc-volume-ds-9 Kernel Manual
 .
 .ds doc-volume-ds-USD   User's Supplementary Documents
 .ds doc-volume-ds-PS1   Programmer's Supplementary Documents
@@ -399,7 +399,7 @@
 .\" NS
 .\" NS override this in `mdoc.local', if necessary
 .
-.ds doc-default-operating-system BSD
+.ds doc-default-operating-system OpenBSD\~ports
 .
 .
 .\" NS doc-operating-system global string
@@ -517,6 +517,10 @@
 .ds doc-operating-system-OpenBSD-4.8  4.8
 .ds doc-operating-system-OpenBSD-4.9  4.9
 .ds doc-operating-system-OpenBSD-5.0  5.0
+.ds doc-operating-system-OpenBSD-5.1  5.1
+.ds doc-operating-system-OpenBSD-5.2  5.2
+.ds doc-operating-system-OpenBSD-5.3  5.3
+.ds doc-operating-system-OpenBSD-5.4  5.4
 .
 .ds doc-operating-system-FreeBSD-1.0     1.0
 .ds doc-operating-system-FreeBSD-1.1     1.1
@@ -732,7 +736,9 @@
 .  ds doc-command-name
 .
 .  ie \n[.$] \{\
-.    ie (\n[.$] == 3) \
+.    ie "\$1"$Mdocdate:" \
+.      ds doc-date-string \$2\~\$3, \$4
+.    el .ie (\n[.$] == 3) \
 .      ds doc-date-string \$1\~\$2 \$3
 .    el \{\
 .      ds doc-date-string "\*[doc-date-\n[mo]]
@@ -906,7 +912,7 @@
 .\" NS width register `Nd' set above
 .
 .de Nd
-.  nop \[em] \$*
+.  nop \[en] \$*
 ..
 .
 .
